### ISSUE TITLE
Buttons not displaying icons on Windows, and some missing button text 

### ISSUE THREAD

**Issue by Tecratal** [Brewtarget/brewtarget#903]
I am using Windows 11 and installed BrewTarget 4.0.11 today. Some of the buttons don't have any text and I have to hover over them and wait for the tool tip to figure out what it does. For example the Add/Remove/Edit fermentable buttons are blank. I attached a screenshot.

![image](https://github.com/user-attachments/assets/ebe2c2ef-16f1-48f6-845f-1e76467d0b0d)

**Comment by matty0ung** [comment]
Sorry you're hitting problems.  This particular problem is actually already fixed in the head revision of the code, and just waiting on the next release (4.0.12 release), which should be available later this week (I hope!).

**Comment by Tecratal** [comment]
Awesome, thank you.

**Comment by matty0ung** [comment]
Hopefully this is fixed in [4.0.12](https://github.com/Brewtarget/brewtarget/releases/tag/v4.0.12).  If not, please don't hesitate to reopen this issue!

**Comment by Tecratal** [comment]
I tried 4.0.12 and it looks like the issue is still there.
![image](https://github.com/user-attachments/assets/fc7a93ed-8d37-4eba-98bc-4c7310ea5cd3)

**Comment by matty0ung** [comment]
Hot dang! Back to the drawing board.

This should have been a small tweak - just getting one extra bit of Qt included in all the packages - but I obviously have missed some subtlety to it.  Will have another look.

**Comment by matty0ung** [comment]
Believe it or not, it's deliberate that some of the buttons only have images.  You can see what it's supposed to look like [here](https://private-user-images.githubusercontent.com/18210330/397051474-00c81d17-df27-4ee9-95e2-25f3383f3ecc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ1NTIzODIsIm5iZiI6MTczNDU1MjA4MiwicGF0aCI6Ii8xODIxMDMzMC8zOTcwNTE0NzQtMDBjODFkMTctZGYyNy00ZWU5LTk1ZTItMjVmMzM4M2YzZWNjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE4VDIwMDEyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTViMDEwZmVjNDU4YTA1ZDQwYzZjMGViMDM3ZjVhOTQxZTA0ZDU0OGZmZjhmMjk3NjVmMTY1M2QwNTE3NjA1NjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.xJDdTq0NDAQtXNX_WscN2uLgp_8gExbsa1xbqdxABLQ).

The issue is that the icons are stored as scalable vector graphics (svg files).  There is a part of the Qt framework that converts them into bitmaps to show on the screen (meaning that they look good on multiple different monitor resolution).  Because of a but in part of the Qt packaging tools, we have to jump through a couple of hoops to ensure that this particular part of the Qt framework ships with the application.  We have it working on Linux (and I hope Mac), but it seems we didn't quite do the right incantations for Windows.

It's a bit hard for me to test on my Windows machine, because it already has all the Qt libraries installed (to do the build).  But I'll keep poking around to try to see what we need to do to get this glitch fixed.

**Comment by matty0ung** [comment]
I'll probably add back-up text to the buttons that are image-only -- not least as it makes the application more accessible.  So we will also make the changes you're suggesting. :thumbsup:

**Comment by matty0ung** [comment]
Original fix was following the notes at https://stackoverflow.com/questions/76047551/icons-shown-in-qt5-not-showing-in-qt6, but I think we might need to do something different because we use MSYS2 and GCC for our Windows build.

**Comment by matty0ung** [comment]
I'm hoping this is now fixed in [4.0.13](https://github.com/Brewtarget/brewtarget/releases/tag/v4.0.13).  (It seemed to work OK on a Windows 10 VM I tried that didn't have all the build environment installed.)  However, as ever, please don't hesitate to reopen this issue if you're still seeing this problem (or to raise a new issue if you hit any other problems).
